organisations/tests/test_api.py

```diff
@@ -909,3 +915,147 @@ def test_person_study_groups_unauthorized(
         variables={"id": to_global_id("PersonNode", person.id)},
     )
     assert executed["data"]["person"]["studygroupSet"]["edges"] == []
+
+
+@pytest.mark.django_db
+class PublicPersonNodeTests:
```

Comment:
Pytest doesn't run these tests, e.g. changing the class name to start with Test made the tests runnable:
```suggestion
class TestPublicPersonNode:
```